plugins/convert_README_to_header.sh

```diff
@@ -30,8 +30,14 @@ outfilename=`basename $outfile`;
 # See http://stackoverflow.com/a/16576291
 # On Mac OS's sed, \n is not recognized as a newline character, but
 # \[actual newline] works
-desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\"/\\\"/g' -e 's/\n/\\\\n"\\
-"/g' "$path/README.md"`;
+desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\\\/\\\\\\\/g' -e 's/\"/\\\"/g'\
+ -e 's/\n/\\\\n\"\
+\"/g' "$path/README.md"`;
+
+if (( $? != 0 )); then
+	echo 'Sed failed to generate $desc'
```

Comment:
Given $desc doesn't really mean anything outside of this script, can we make this more generic, say "Sed failed to process readme" or something?